### PR TITLE
Edit a child with basic UI

### PR DESCRIPTION
**Pull request by claudianguyen**

### CODE REVIEW

File: src/main/java/com/tucklets/app/controllers/AdminController.java [hunk 1]
```diff
@@ -43,6 +46,27 @@ public String upload() {
         return "admin/upload-children-data";
     }
 
+
```
Comment (ShahilZ) [inline src/main/java/com/tucklets/app/controllers/AdminController.java:49]: Extra line?

File: src/main/resources/templates/admin/dashboard.html [hunk 2]
```diff
@@ -52,7 +56,55 @@ <h4>Children Information</h4>
         <a th:href="@{/admin/dashboard/upload}" class="btn btn-primary upload-children-data-button">Upload Children Data</a>
     </div>
 
+    <div class="">
+        <form class="edit-child-form" th:action="@{/admin/dashboard/edit-child}" th:object="${child}" method="post">
```
Comment (ShahilZ) [inline src/main/resources/templates/admin/dashboard.html:60]: Let's change "edit" --> "modify"

Comment (claudianguyen) [reply src/main/resources/templates/admin/dashboard.html]: done

File: src/main/java/com/tucklets/app/controllers/AdminController.java (hunk 1 above)
Comment (claudianguyen) [reply src/main/java/com/tucklets/app/controllers/AdminController.java:51]: done

**Pull request merged**